.github/workflows/fuzzing-build.yaml

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Fuzzing Builds
+
+on:
+    # For now, only manual triggers.
+    workflow_dispatch:
+
+concurrency:
+    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    build_linux_fuzzing:
+        name: Build on Linux
+        timeout-minutes: 90
+
+        runs-on: ubuntu-latest
+        if: github.actor != 'restyled-io[bot]'
+
+        container:
+            image: connectedhomeip/chip-build:0.5.48
+            volumes:
+                - "/tmp/log_output:/tmp/test_logs"
+            options:
+                --sysctl "net.ipv6.conf.all.disable_ipv6=0
+                net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1"
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  submodules: true
+            - run: apt-get update
+            - run: apt-get install --fix-missing llvm-10 clang-10
```

Comment:
I think these should be part of the base image. Doing this in every build seems painful.

Reply:
Note that this job does not run unless manually triggered.  I am happy to get the base image updated, though; I'll prod at that.

Reply:
@andy31415 Should these be part of the Matter base image, or should we have an image for this job specifically?

I'd prefer to do this update in a separate PR, by the way, since this job does not run automatically yet... but in the meantime people can start using it as needed.

Reply:
Ok, did not notice "manual trigger".

I believe enventually we  should udpate our images to have all tools. E.g. I likely want to be able to just run fuzzing in my vscode image.